src/ontology/cl-edit.owl

```diff
@@ -32898,6 +32899,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18621753") rdfs:comment
 AnnotationAssertion(rdfs:label obo:CL_4052022 "fallopian tube smooth muscle cell")
 EquivalentClasses(obo:CL_4052022 ObjectIntersectionOf(obo:CL_0000192 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003889)))
 
+# Class: obo:CL_4052023 (luminal endometrial unciliated epithelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36581776") Annotation(oboInOwl:hasDbXref "PMID:38559249") Annotation(oboInOwl:hasDbXref "PMID:39198675") Annotation(oboInOwl:hasDbXref "PMID:32929266") obo:IAO_0000115 obo:CL_4052023 "An epithelial cell that is part of the endometrial luminal epithelium, forming a continuous layer lining the uterine cavity. This cell undergoes cyclical changes during the menstrual cycle, proliferating under estrogen in the proliferative phase, and differentiating under progesterone in the secretory phase to prepare for potential implantation. During the window of implantation, the cell changes from a tall columnar shape to a shorter columnar or cuboidal form, loses polarity, and becomes receptive to blastocyst implantation.")
```

Comment:
I suggest to change from the cell to this cell:

"During the window of implantation, `this` cell changes from a tall columnar shape"